docs/history/changelog.md

```diff
@@ -18,6 +18,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
     * Add examples [#1338](https://github.com/open-contracting/standard/pull/1338):
       * Organization references [#947](https://github.com/open-contracting/standard/pull/947)
       * Organization identifiers [#948](https://github.com/open-contracting/standard/pull/948)
+        [#1429](https://github.com/open-contracting/standard/pull/1429)
```

Comment:
I think this part of the changelog is about the addition of new examples.
```suggestion
```

Reply:
To avoid a very long changelog with less relevant bullets, it's appropriate to put small updates to existing worked examples alongside the example's original entry.